groups/bdl/bdlb/bdlb_guid.h

```diff
@@ -517,6 +521,13 @@ unsigned char Guid::version() const {
     return uc(timeHiAndVersion() >> 12);
 }
 
+template <class HASH_ALGORITHM>
```

Comment:
I'll favour readability and stick to `HASH_ALGORITHM` (see also usage of `TYPE` in BDE over `T` as is more common outside BDE) unless there is a strong objection.